ConsentViewController/Classes/SourcePointClient/SourcePointClient.swift

```diff
@@ -182,7 +182,9 @@ class SourcePointClient: SourcePointProtocol {
                 requestUUID: requestUUID,
                 pmSaveAndExitVariables: pmPayload,
                 meta: meta,
-                publisherData: action.publisherData
+                publisherData: action.publisherData,
+                consentLanguage: (action.consentLanguage != nil) ?
+                    action.consentLanguage : Locale.preferredLanguages[0].uppercased()
```

Comment:
@vilasma I think we shouldn't put the responsibility to fill in the default language `Locale.preferredLanguages[0].uppercased()` in the `SPClient`. 
This should be set as soon as the `Action` is created.

Reply:
@andresilveirah added Locale value in once action is triggered from native message